### PR TITLE
feat(Animeler.net): New presence

### PR DESCRIPTION
**Pull request by Chasology**
## Description 
Animeler.net is a turkish anime news website.

* Detects homepage.
* Detects article headline.
* Detects user profile.
* Detects article tags.

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Proof 1

Proof 2
</details>

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Please run `yarn format` and resolve SV CI

File: websites/A/Animeler.net/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Chas",
+		"id": "498195150711947285"
+	},
+	"service": "Animeler.net",
+	"description": {
+		"en": "Animeler.net is a website that you can reach latest news about animes. Provided by Turk Anime TV.",
+		"tr": "Animeler.net, anime haberlerini size en hızlı şekilde sunan sitedir. Türk Anime TV iştirakidir.",
```
Comment (EncryptedDev) [inline websites/A/Animeler.net/dist/metadata.json:10]: This is the official description in Turkish

```suggestion
		"tr": "En iyi, en güncel, en çok konuşulan içerikler ve farklı kaynaklardan tamamen tarafsız içerikleri okuyun. Türk Anime TV iştirakidir.",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/A/Animeler.net/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		"nl": "Animeler.net is de site die u op de snelste manier anime-nieuws aanbiedt. Het is een dochteronderneming van Turk Anime TV."
+	},
+	"url": "www.animeler.net",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/9JPsUmz.png",
```
Comment (EncryptedDev) [inline websites/A/Animeler.net/dist/metadata.json:15]: Not 512 x 512

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Sorry, this was suppose to be bundled with the other review

File: websites/A/Animeler.net/dist/metadata.json (hunk 2 above)
Comment (Chasology) [reply websites/A/Animeler.net/dist/metadata.json]: fixed

File: websites/A/Animeler.net/presence.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	if (desc) {
+		presenceData.details = desc;
+		presenceData.state = headline;
+	}
+
+	if (presenceData.details) {
+		presenceData.buttons = [
+			{
+				label: "Sayfaya git",
+				url: document.location.href.toString()
```
Comment (EncryptedDev) [inline websites/A/Animeler.net/presence.ts:63]:
```suggestion
				url: document.location.href
```

File: websites/A/Animeler.net/presence.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+			} as ButtonData
```
Comment (EncryptedDev) [inline websites/A/Animeler.net/presence.ts:64]:
```suggestion
			}
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by eggsy — CHANGES_REQUESTED** [review]
Strings are fine, please resolve everything @EncryptedDev mentioned.

**Review by eggsy — APPROVED** [review]

**Comment by EncryptedDev** [comment]
This pull request will be closed in the next few days if the my review isn't addressed fully

**Comment by EncryptedDev** [comment]
Inactive

**Pull request closed without merging**